tests/pipeline/nodes/output/test_media_writer.py

```diff
@@ -48,7 +49,7 @@ def test_writer_writes_single_image(self, writer, create_image):
         image = create_image(size)
         writer.run({"filename": "test.jpg", "img": image, "fps": 1})
         assert directory_contents() == set(["test.jpg"])
-
+        
     def test_writer_writes_multi_image(self, writer, create_image):
```

Comment:
Some of the tests are not passing, it will be good to fix them within the issue.

Reply:
temporarily shutil.rmtree(newpath, ignore_errors=True) is used to pass the pytest. subsequent check of the root permission error have to be done in future

Reply:
ok thanks